### ISSUE TITLE
Test failure due to missing ZWARN bit

### ISSUE THREAD

**Issue by weaverba137** [desihub/desitarget#762]
Nightly integration tests on desitarget are failing with this message:
```
======================================================================
FAIL: test_zwarn_in_sync (desitarget.test.test_mtl.TestMTL)
Check redrock and desitarget ZWARN bit-masks are consistent.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/global/common/software/desi/cori/desiconda/20200801-1.4.0-spec/code/desitarget/master/py/desitarget/test/test_mtl.py", line 401, in test_zwarn_in_sync
    self.assertTrue(bitname in dtbitnames,
AssertionError: False is not true : missing ZWARN bit POORDATA from redrock

----------------------------------------------------------------------
Ran 87 tests in 475.860s
```

**Comment by geordie666** [comment]
I believe this is being addressed by Stephen in #757.

**Comment by geordie666** [comment]
@sbailey: I think you addressed this in #757. Feel free to reopen this issue if I'm incorrect on that point.
